TabulateSmarterTestPackage/Tabulators/ItemTabulator.cs

```diff
@@ -371,9 +370,9 @@ public ItemTabulator()
                 var j = 0;
                 foreach (var p in performanceLevels)
                 {
-                    itemFields[ItemFieldNamesCount + j++] = p.PerfLevel;
-                    itemFields[ItemFieldNamesCount + j++] = p.ScaledLow;
-                    itemFields[ItemFieldNamesCount + j++] = p.ScaledHigh;
+                    itemFields[ItemFieldNamesCount - 3 * performanceLevels.Count + j++] = p.PerfLevel;
```

Comment:
We don't know in advance how many cutpoints there are, so I'm generating the column labels for them dynamically as needed. The cutpoints and their scale were being offset off of the end of their column labels. This puts them back in line with their labels.